clients/client-rust/src/lib.rs

```diff
@@ -0,0 +1,275 @@
+/*!
+# Taskcluster Client for Rust
+
+For a general guide to using Taskcluster clients, see [Calling Taskcluster
+APIs](https://docs.taskcluster.net/docs/manual/using/api).
+
+This client is a convenience wrapper around `reqwest` that provides named functions for each API
```

Comment:
``reqwest`` is a better name for SEO than Python's ``requests``, and so much better than go's ``got``. +1 to Rust.

Reply:
but so hard to type!